src/Simplex/Messaging/Agent.hs

```diff
@@ -46,13 +47,17 @@ runSMPAgent cfg@AgentConfig {tcpPort} = do
       q <- asks $ tbqSize . config
       n <- asks clientCounter
       c <- atomically $ newAgentClient n q
-      logInfo $ "client " <> showText (clientId c) <> " connected to Agent"
+      logConnection c True
```

Comment:
Is it possible to create synonyms for data constructors? Doesn't seem to be from what I'm finding..
I.e. `Connected` = `True`, `Disconnected` = `False`

Or maybe create a type to make intension more explicit? E.g. `data ConnectionStatus = Connected | Disconnected`

Reply:
Nvm I guess, I'm probably overthinking it